elements/pfe-sass/extends/_extends.scss

```diff
@@ -56,3 +56,84 @@
     color: pfe-fetch(text) !important;
   }
 }
+
+%ordered-list-primary {
+  // size of the font inside of the circle
+  --FontSize: var(--pfe-list-FontSize, 24px);
```

Comment:
Can you source the fallback values from the maps or using `pfe-var` if possible?

Reply:
Replaced with $pf-global--FontSize--2xl